src/Solver.cpp

```diff
@@ -298,6 +298,17 @@ FP_PRECISION Solver::getFlux(int fsr_id, int group) {
 
 
 /**
+ * @brief Returns the boundary flux array at the requested indexes
```

Comment:
actually, "flux array at the requested indexes" might be more clear if explained as "flux array for some track"